zerver/migrations/0479_realm_backfill_uuid_and_secret.py

```diff
@@ -0,0 +1,67 @@
+import secrets
+import uuid
+
+from django.db import migrations, models
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def generate_api_key() -> str:
+    """
+    This is a copy of zerver.lib.utils.generate_api_key
```

Comment:
It may be worth expanding this as to why we don't do this as an import, if the subsequent migration does so.  My understanding of why is that we want to be consistent about what version of `generate_api_key` we were using at this point, even if that function drifts in future commits -- which isn't relevant in the follow-up migration.

Reply:
Yes, essentially. Importing here probably would work fine in this particular case, but we just avoid that by habit and have the lint rule against it. The last migration specifies the `default=` - and this is the typical case where we make an exception from that and just import (you can see if you `git grep` for those other `pattern-not`s in the `dont-import-models-in-migrations` semgrep rule.